### DISCUSSION TITLE
Error while installing winter cms

### DISCUSSION THREAD

**Discussion by Anshul-code** [wintercms/winter#928] [General]
![image](https://github.com/wintercms/winter/assets/76148784/206d3a82-272a-4408-8fbb-aede9d56f29c)

Did I do something wrong ?

I ran following:
`composer create-project wintercms/winter newwinapp "dev-develop"`

**Comment by bennothommo** [comment]
Hi @Anshul-code, the source manifest may not have been available at the time, as it is a file hosted on GitHub. If you run `php artisan winter:version` again, does it fail?

**Comment by Anshul-code** [comment]
![image](https://github.com/wintercms/winter/assets/76148784/c3f0016a-67e5-41c5-897d-ea4e773d9542)
I installed new project using wintercms again now. It works.
I don't know why it was not running before.
Can you give me insight on this, Why it was not working before ?

**Reply by bennothommo** [reply]
@Anshul-code could've been an error with retrieving the file from GitHub, or it could've been an intermittent internet issue.

**Comment by LukeTowers** [comment]
@bennothommo we should probably improve the error handling there to just provide a warning about the address being unreachable and to try again later instead of erroring out at that point.
